src/core/sphinx/build.rs

```diff
@@ -234,7 +225,7 @@ pub fn complete_request_packet(
 }
 
 /// Size in bytes of a [`Surb`].
-pub const SURB_SIZE: usize = RAW_MIXNODE_INDEX_SIZE + HEADER_SIZE + PAYLOAD_ENCRYPTION_KEY_SIZE;
+pub const SURB_SIZE: usize = RAW_MIXNODE_INDEX_SIZE + HEADER_SIZE + SHARED_SECRET_SIZE;
```

Comment:
IIUC the size did change here, do we have a tests or debug_assertions that ensure that surb packet is same size as non surb packet?

Reply:
This is the size of a SURB, _not_ a SURB packet. Packets constructed from SURBs are no different to normal packets really, they just have a deliver-reply action at the end instead of eg a deliver-request action.

Reply:
ok, guess the actual packet size is calculated this constant as parameter. Just was mentioning it as it can be really good to have such assertion (but may be they are already in place, I did not check).

Reply:
SURBs are passed in fragment payloads. Fragment payloads are a fixed size, which is independent of `SURB_SIZE`. `SURB_SIZE` just affects eg how many SURBs can fit in a single fragment.

Reply:
oh yes :+1:, I remember when reading the spec last time that there could be larger number of surbs than the message payload by putting surbs in multiple fragements, and yet it did not click that this comment was totally wrong.